### PR TITLE
Fix the Hello_World example

### PR DESCRIPTION
**Pull request by rahulporuri**
The fix is fairly trivial - it is replacing `logging.StreamHandler` with `logging.FileHandler`.

### CODE REVIEW

**Comment by kitchoi** [comment]
Appveyor build failed with this:
```
[EXECUTING] C:\Users\appveyor\.edm\envs\edm\Scripts\edm.EXE install -y -e envisage-test-3.6-pyqt5 traits pyqt5 coverage enthought_sphinx_theme traitsui ipykernel flake8 pyface apptools sphinx
Fetching indices for package repositories.. done
All Fetching package harfbuzz attempts failed
HTTPSConnectionPool(host='package-data.enthought.com', port=443): Max retries exceeded with url: 
...
```

**Comment by rahulporuri** [comment]
I restarted the PR. It seems to be an intermittent failure. [log(2).txt](https://github.com/enthought/envisage/files/5407970/log.2.txt)

File: examples/Hello_World/hello_world.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 # Create a log file.
 logger = logging.getLogger()
-logger.addHandler(logging.StreamHandler(file("hello_world.log", "w")))
+logger.addHandler(logging.StreamHandler(open("hello_world.log", "w")))
```
Comment (mdickinson) [inline examples/Hello_World/hello_world.py:14]: Similar to @kitchoi 's comment on #319, we probably want `FileHandler` here.

If the `open` stays, please add the encoding parameter. As a general rule, all uses of `open` should have that parameter explicitly specified, else potentially odd system-dependent things happen on Windows.

**Review by kitchoi — APPROVED** [review]
LGTM

File: examples/Hello_World/hello_world.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 # Create a log file.
 logger = logging.getLogger()
-logger.addHandler(logging.StreamHandler(file("hello_world.log", "w")))
+logger.addHandler(logging.FileHandler("hello_world.log", "w", encoding="utf8"))
```
Comment (kitchoi) [inline examples/Hello_World/hello_world.py:14]: nit: "u8"? "utf-8"? "utf8"? 😂

Comment (rahulporuri) [reply examples/Hello_World/hello_world.py:14]: sigh. looks like the rest of the codebase uses `utf-8`. fixing.

Comment (mdickinson) [reply examples/Hello_World/hello_world.py]: `utf-8` is the official name of the Python codec (see the exception message below for an example); the other variants are aliases. (UTF-8 is the official Unicode spelling.)
```
>>> b'\x80abc'.decode('utf8')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x80 in position 0: invalid start byte
```

**Pull request merged**